app/templates/_widget.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference path="../../../tsd.ts.ts" />
+/// <amd-dependency path="dojo/text!./templates/<%=widgetName%>.html" name="template" />
+declare var template: any;
+
+//dojo
+import dojoDeclare = require("dojo/_base/declare");
+
+//dijit
+import WidgetBase = require("dijit/_WidgetBase");
+import TemplatedMixin = require("dijit/_TemplatedMixin"); 
+<% if(widgetsInTemplate) { %>import WidgetsInTemplateMixin = require("dijit/_WidgetsInTemplateMixin"); <% }%>
+
+interface WidgetInterface {
```

Comment:
I wonder if this interface should be in it's own module and imported into this one so that it's not repeated in every widget?